crates/rethnet_eth/src/remote/client.rs

```diff
@@ -415,6 +415,11 @@ impl RpcClient {
         self.call(&MethodInvocation::GetStorageAt(*address, position, block))
             .await
     }
+
+    /// Calls `net_version` API
+    pub async fn network_id(&self) -> Result<U256, RpcClientError> {
```

Comment:
```suggestion
    pub async fn net_version(&self) -> Result<U256, RpcClientError> {
```

So far here the convention has been to name these associated methods according to the method being invoked (at least for the methods that are really just wrappers around RPC methods (`get_account_info` being the exception)).